Timestamps all messages with the method used to timestamp messages that show up late

Modified for 1.2 by StoneRyno

Index: /usr/palm/applications/com.palm.app.messaging/app/controllers/chatview-assistant.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.messaging/app/controllers/chatview-assistant.js
+++ /usr/palm/applications/com.palm.app.messaging/app/controllers/chatview-assistant.js
@@ -1303,7 +1303,7 @@ var ChatviewAssistant = Class.create({
 	
 			// if the INCOMING message was sent longer than 10 minutes before it arrived on device,
 			// add sent date to indicate that this delay happened
-			if(!ChatFlags.isOutgoing(msg.flags) && msg.deviceTimeStamp && msg.timeStamp && (msg.deviceTimeStamp-msg.timeStamp) > (1000*60*10)) {
+//			if(!ChatFlags.isOutgoing(msg.flags) && msg.deviceTimeStamp && msg.timeStamp && (msg.deviceTimeStamp-msg.timeStamp) > (1000*60*10)) {
 				var d = new Date();
 				var today = d.getTime();
 				d.setTime(msg.timeStamp);
@@ -1311,10 +1311,10 @@ var ChatviewAssistant = Class.create({
 				
 				// Only show the message sent time if it is less than two days old
 				// from when it arrived on device
-				if(today-msg.timeStamp <= (1000*60*60*24*2)) {
+//				if(today-msg.timeStamp <= (1000*60*60*24*2)) {
 					msg.timeStampDeltaMsg = new Template($L("Message Sent: #{dateTime}")).evaluate({dateTime:Mojo.Format.formatDate(sentTimeStamp,'short')});
-				}		
-			}
+//				}		
+//			}
 
 			if (msg.callbackNumber) {
 				msg.callbackNumber = PalmSystem.runTextIndexer(msg.callbackNumber);
